DependencyInjection/Configuration.php

```diff
@@ -75,6 +75,19 @@ public function getConfigTreeBuilder()
                     ->info('If null, the user ID claim will have the same name as the one defined by the option "user_identity_field"')
                 ->end()
                 ->append($this->getTokenExtractorsNode())
+                ->arrayNode('set_cookie')
```

Comment:
Could it be possible to set several cookies, or to configure the name per firewall? For instance Mercure requires to set a cookie containing a JWT names `mercureAuthorization`, but the app or other subsystems could need another token stored in another cookie.

Reply:
Node renamed to `set_cookies` which now takes an arbitrary number of cookies indexed by name, accepting the cookie lifetime, path, domain and samesite attributes as options.
Adding a `firewall` option in another PR is possible, in case setting the cookie path is not enough. Thanks!